<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>SMUControlWidget</class>
 <widget class="QWidget" name="SMUControlWidget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1488</width>
    <height>683</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QFormLayout" name="formLayout">
   <item row="0" column="0">
    <layout class="QGridLayout" name="gridLayout_3">
     <item row="1" column="0">
      <layout class="QGridLayout" name="gridLayout">
       <item row="0" column="0">
        <spacer name="horizontalSpacer">
         <property name="orientation">
          <enum>Qt::Horizontal</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>498</width>
           <height>20</height>
          </size>
         </property>
        </spacer>
       </item>
       <item row="0" column="1">
        <widget class="QPushButton" name="resetInstrBtn">
         <property name="text">
          <string>Reset</string>
         </property>
         <property name="icon">
          <iconset resource="../materials.qrc">
           <normaloff>:/materials/materials/reset.png</normaloff>:/materials/materials/reset.png</iconset>
         </property>
        </widget>
       </item>
      </layout>
     </item>
     <item row="2" column="0">
      <widget class="QGroupBox" name="groupBox">
       <property name="maximumSize">
        <size>
         <width>600</width>
         <height>16777215</height>
        </size>
       </property>
       <property name="title">
        <string>Measurements</string>
       </property>
       <layout class="QGridLayout" name="gridLayout_7">
        <item row="0" column="1">
         <widget class="QGroupBox" name="groupBox_3">
          <property name="layoutDirection">
           <enum>Qt::LeftToRight</enum>
          </property>
          <property name="autoFillBackground">
           <bool>true</bool>
          </property>
          <property name="title">
           <string>SMU B</string>
          </property>
          <property name="alignment">
           <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignVCenter</set>
          </property>
          <property name="flat">
           <bool>false</bool>
          </property>
          <property name="checkable">
           <bool>true</bool>
          </property>
          <property name="checked">
           <bool>false</bool>
          </property>
          <layout class="QGridLayout" name="gridLayout_6">
           <item row="3" column="4">
            <widget class="QLabel" name="smub_p_2">
             <property name="maximumSize">
              <size>
               <width>200</width>
               <height>50</height>
              </size>
             </property>
             <property name="font">
              <font>
               <family>Source Code Pro</family>
               <pointsize>16</pointsize>
               <bold>true</bold>
              </font>
             </property>
             <property name="text">
              <string>0.0</string>
             </property>
             <property name="alignment">
              <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
             </property>
            </widget>
           </item>
           <item row="1" column="1">
            <widget class="QLabel" name="smub_v_2">
             <property name="maximumSize">
              <size>
               <width>200</width>
               <height>50</height>
              </size>
             </property>
             <property name="font">
              <font>
               <family>Source Code Pro</family>
               <pointsize>16</pointsize>
               <bold>true</bold>
              </font>
             </property>
             <property name="text">
              <string>0.0</string>
             </property>
             <property name="alignment">
              <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
             </property>
            </widget>
           </item>
           <item row="3" column="0" colspan="2">
            <widget class="QLabel" name="smub_r_2">
             <property name="maximumSize">
              <size>
               <width>200</width>
               <height>50</height>
              </size>
             </property>
             <property name="font">
              <font>
               <family>Source Code Pro</family>
               <pointsize>16</pointsize>
               <bold>true</bold>
              </font>
             </property>
             <property name="text">
              <string>0.0</string>
             </property>
             <property name="alignment">
              <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
             </property>
            </widget>
           </item>
           <item row="1" column="5">
            <widget class="QLabel" name="label_32">
             <property name="text">
              <string>A</string>
             </property>
            </widget>
           </item>
           <item row="3" column="5">
            <widget class="QLabel" name="label_30">
             <property name="text">
              <string>W</string>
             </property>
            </widget>
           </item>
           <item row="2" column="3">
            <widget class="QLabel" name="label_27">
             <property name="text">
              <string>Power</string>
             </property>
            </widget>
           </item>
           <item row="1" column="4">
            <widget class="QLabel" name="smub_c_2">
             <property name="maximumSize">
              <size>
               <width>200</width>
               <height>50</height>
              </size>
             </property>
             <property name="font">
              <font>
               <family>Source Code Pro</family>
               <pointsize>16</pointsize>
               <bold>true</bold>
              </font>
             </property>
             <property name="text">
              <string>0.0</string>
             </property>
             <property name="alignment">
              <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
             </property>
            </widget>
           </item>
           <item row="0" column="3">
            <widget class="QLabel" name="label_28">
             <property name="text">
              <string>Current</string>
             </property>
            </widget>
           </item>
           <item row="2" column="0">
            <widget class="QLabel" name="label_29">
             <property name="text">
              <string>Resistance</string>
             </property>
            </widget>
           </item>
           <item row="0" column="0">
            <widget class="QLabel" name="label_25">
             <property name="text">
              <string>Voltage</string>
             </property>
            </widget>
           </item>
           <item row="1" column="2">
            <widget class="QLabel" name="label_26">
             <property name="text">
              <string>V</string>
             </property>
            </widget>
           </item>
           <item row="3" column="2">
            <widget class="QLabel" name="label_31">
             <property name="text">
              <string>Ohm</string>
             </property>
            </widget>
           </item>
          </layout>
         </widget>
        </item>
        <item row="0" column="0">
         <widget class="QGroupBox" name="groupBox_2">
          <property name="layoutDirection">
           <enum>Qt::LeftToRight</enum>
          </property>
          <property name="autoFillBackground">
           <bool>true</bool>
          </property>
          <property name="title">
           <string>SMU A</string>
          </property>
          <property name="flat">
           <bool>true</bool>
          </property>
          <property name="checkable">
           <bool>true</bool>
          </property>
          <property name="checked">
           <bool>false</bool>
          </property>
          <layout class="QGridLayout" name="gridLayout_5">
           <item row="3" column="2">
            <widget class="QLabel" name="label_15">
             <property name="text">
              <string>Ohm</string>
             </property>
            </widget>
           </item>
           <item row="0" column="3">
            <widget class="QLabel" name="label_5">
             <property name="text">
              <string>Current</string>
             </property>
            </widget>
           </item>
           <item row="1" column="5">
            <widget class="QLabel" name="label_17">
             <property name="text">
              <string>A</string>
             </property>
            </widget>
           </item>
           <item row="1" column="4">
            <widget class="QLabel" name="smua_c_2">
             <property name="maximumSize">
              <size>
               <width>200</width>
               <height>50</height>
              </size>
             </property>
             <property name="font">
              <font>
               <family>Source Code Pro</family>
               <pointsize>16</pointsize>
               <bold>true</bold>
              </font>
             </property>
             <property name="text">
              <string>0.0</string>
             </property>
             <property name="alignment">
              <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
             </property>
            </widget>
           </item>
           <item row="3" column="5">
            <widget class="QLabel" name="label_21">
             <property name="text">
              <string>W</string>
             </property>
            </widget>
           </item>
           <item row="3" column="0" colspan="2">
            <widget class="QLabel" name="smua_r_2">
             <property name="maximumSize">
              <size>
               <width>200</width>
               <height>50</height>
              </size>
             </property>
             <property name="font">
              <font>
               <family>Source Code Pro</family>
               <pointsize>16</pointsize>
               <bold>true</bold>
              </font>
             </property>
             <property name="text">
              <string>0.0</string>
             </property>
             <property name="alignment">
              <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
             </property>
            </widget>
           </item>
           <item row="0" column="0">
            <widget class="QLabel" name="label_6">
             <property name="text">
              <string>Voltage</string>
             </property>
            </widget>
           </item>
           <item row="2" column="3">
            <widget class="QLabel" name="label_7">
             <property name="text">
              <string>Power</string>
             </property>
            </widget>
           </item>
           <item row="1" column="1">
            <widget class="QLabel" name="smua_v_2">
             <property name="maximumSize">
              <size>
               <width>200</width>
               <height>50</height>
              </size>
             </property>
             <property name="font">
              <font>
               <family>Source Code Pro</family>
               <pointsize>16</pointsize>
               <bold>true</bold>
              </font>
             </property>
             <property name="text">
              <string>0.0</string>
             </property>
             <property name="alignment">
              <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
             </property>
            </widget>
           </item>
           <item row="1" column="2">
            <widget class="QLabel" name="label_23">
             <property name="text">
              <string>V</string>
             </property>
            </widget>
           </item>
           <item row="3" column="4">
            <widget class="QLabel" name="smua_p_2">
             <property name="maximumSize">
              <size>
               <width>200</width>
               <height>50</height>
              </size>
             </property>
             <property name="font">
              <font>
               <family>Source Code Pro</family>
               <pointsize>16</pointsize>
               <bold>true</bold>
              </font>
             </property>
             <property name="text">
              <string>0.0</string>
             </property>
             <property name="alignment">
              <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
             </property>
            </widget>
           </item>
           <item row="2" column="0">
            <widget class="QLabel" name="label_8">
             <property name="text">
              <string>Resistance</string>
             </property>
            </widget>
           </item>
          </layout>
         </widget>
        </item>
       </layout>
      </widget>
     </item>
     <item row="0" column="0">
      <widget class="QTabWidget" name="tabWidget">
       <property name="maximumSize">
        <size>
         <width>600</width>
         <height>16777215</height>
        </size>
       </property>
       <property name="focusPolicy">
        <enum>Qt::NoFocus</enum>
       </property>
       <property name="currentIndex">
        <number>0</number>
       </property>
       <property name="documentMode">
        <bool>false</bool>
       </property>
       <widget class="QWidget" name="smua">
        <attribute name="icon">
         <iconset resource="../materials.qrc">
          <normaloff>:/materials/materials/mosfet.png</normaloff>:/materials/materials/mosfet.png</iconset>
        </attribute>
        <attribute name="title">
         <string>SMU A</string>
        </attribute>
       </widget>
       <widget class="QWidget" name="smub">
        <attribute name="icon">
         <iconset resource="../materials.qrc">
          <normaloff>:/materials/materials/font.png</normaloff>:/materials/materials/font.png</iconset>
        </attribute>
        <attribute name="title">
         <string>SMU B</string>
        </attribute>
        <layout class="QGridLayout" name="gridLayout_2"/>
       </widget>
      </widget>
     </item>
    </layout>
   </item>
   <item row="0" column="1">
    <layout class="QVBoxLayout" name="graphLayout"/>
   </item>
  </layout>
 </widget>
 <resources>
  <include location="../materials.qrc"/>
 </resources>
 <connections/>
</ui>
